scripts/ci/check_compliance.py

```diff
@@ -1019,6 +1020,7 @@ def check_no_undef_outside_kconfig(self, kconf):
         "BOARD_MPS2_AN521_CPUTEST", # Used for board and SoC extension feature tests
         "BOARD_NATIVE_SIM_NATIVE_64_TWO", # Used for board and SoC extension feature tests
         "BOARD_NATIVE_SIM_NATIVE_ONE", # Used for board and SoC extension feature tests
+        "BOARD_REVISION_",  # Used for automating boolean board revision options
```

Comment:
Hmm... this can actually be removed. I thought at first this script would use the prefix and append a catch-anything pattern after it, but that is not the case. Hence the `sym_name.startswith` above.

Reply:
Fixed!